audformat/core/column.py

```diff
@@ -149,11 +154,24 @@ def get(
                 assigned to a scheme that contains a dict mapping
                 speaker IDs to age entries, ``map='age'``
                 will replace the ID values with the age of the speaker
+            as_segmented: always return a segmented index
```

Comment:
I think we need to be more specific here (and in the table documentation at the same place) what will happen. Maybe:

```suggestion
            as_segmented: always return a segmented index.
                ``start`` will be set to ``0``.
                ``end`` to ``NaT`` or to the duration of the file
                if ``allow_nat`` is set to ``False``
```

Reply:
What about:

![image](https://user-images.githubusercontent.com/10383417/141081673-f34fcd8a-b143-4445-8587-5e571d73d5f6.png)

Reply:
Yes, perfect.

Reply:
Maybe instead of `labels` we use `table` and `column`?

Reply:
I now changed to:

![image](https://user-images.githubusercontent.com/10383417/141085955-628dd88b-a066-4ca5-891c-774667910239.png)

and

![image](https://user-images.githubusercontent.com/10383417/141086029-7d12d33c-e960-4dbb-943f-915370398e4d.png)